src/temp/Project/Search.module.scss

```diff
@@ -0,0 +1,203 @@
+.searchBarMainConatiner {
+  background-color: #fff;
+  width: 338px;
```

Comment:
Why is the width hardcoded? This affects responsiveness.

Reply:
Most of the width will be replaced by 100% later, as it would take the whole width of the sidebar, so the container widths are temporary as of now. @sunilsabatp could you add a comment here?